toot/console.py

```diff
@@ -297,6 +297,10 @@ def editor(value):
                 "help": "path to the media file to attach (specify multiple "
                         "times to attach up to 4 files)"
             }),
+            (["-d", "--description"], {
+                "type": str,
+                "help": 'A plain-text description of the media, for accessibility purposes.'
+            }),
```

Comment:
One issue: since toot supports multiple `--media` options, it also needs to support multiple `--description` options.